model/riscv_vext_control.sail

```diff
@@ -8,6 +8,14 @@
 
 function clause extensionEnabled(Ext_V) = (misa[V] == 0b1) & (mstatus[VS] != 0b00)
 
+// Note: Current implementation does not trap when vstart is out of bounds
```

Comment:
Probably should say that trapping is recommended but not required.

```suggestion
// Note: The spec recommends trapping if vstart is out of bounds but the
// current implementation does not do that because write_CSR() doesn't
// allow traps.
```
```